org.thingpedia.iot.thermostat/dataset.tt

```diff
@@ -0,0 +1,130 @@
+dataset @org.thingpedia.iot.thermostat language "en" {
+
+    program := now => @org.thingpedia.iot.thermostat.humidity() => notify
+    #_[utterances=["is it humid?",
+                   "check if it is humid",
+                   "what does my humidity sensor say?",
+                   "how humid is it?",
+                   "what is the level of humidity?",
+                   "how high is the humidity?"]];
+
+    stream := monitor @org.thingpedia.iot.thermostat.humidity()
+    #_[utterances=["when the humidity changes"]];
+
+    stream (p_state : Enum(humid, normal)) := edge( monitor @org.thingpedia.iot.thermostat.humidity()) on (state == p_state)
+    #_[utterances=["when the humidity sensor says it is ${p_state}"]];
+
+    stream := edge( monitor @org.thingpedia.iot.thermostat.humidity()) on (state == enum(humid))
+    #_[utterances=["when it gets humid",
+                   "when the humidity is high",
+                   "when it gets very humid",
+                   "when it starts getting super humid",
+                   "when it is too humid in the area"]];
+
+    stream := edge( monitor @org.thingpedia.iot.thermostat.humidity()) on (state == enum(normal))
+    #_[utterances=["when the humidity is normal",
+                   "when it is no longer humid",
+                   "when it is not humid anymore"]];
+
+    program := now => @org.thingpedia.iot.thermostat.temperature() => notify
+    #_[utterances=["is the indoor temperature warm?",
+                   "check if the indoor temperature is warm",
+                   "is the indoor temperature cold?",
+                   "check if the room temperature is warm or cold",
+                   "what is the indoor temperature?",
+                   "how high is the indoor temperature?"]];
+
+    query := @org.thingpedia.iot.thermostat.temperature()
+    #_[utterances=["the state of the room temperature",
+                   "the room temperature",
+                   "how high the room temperature is",
+                   "if the inside temperature is warm or cold",
+                   "whether the room temperature is warm or cold"]];
```

Comment:
Similarly, we already have a temperature sensor.

Reply:
Fixed in latest commit